client/view/Form/Form.css

```
@@ -73,24 +74,55 @@
   color: var(--error);
 }
 
-.Form--serverError .Form__hint--error {
-  opacity: 100%;
-}
-
-.Form--serverError .Form__hint--error a {
+.Form__hint--error a {
   color: inherit;
   pointer-events: initial;
 }
 
+.Form.Form--serverError:not(.Form--loaded) .Form__hint--error {
+  opacity: 100%;
+}
+
 .Form__queryTextArea:placeholder-shown ~ .Form__interactivePlaceholder {
+  pointer-events: none;
+  opacity: 100%;
+}
+
+.Form__queryTextArea:placeholder-shown ~ .Form__interactivePlaceholder a {
   pointer-events: initial;
+}
+
+.Form:not(.Form--serverError):not(.Form--loaded) .Form__queryTextArea:invalid:focus-visible ~ .Form__hint--required {
   opacity: 100%;
 }
 
-.Form:not(.Form--serverError) .Form__queryTextArea:invalid:focus-visible ~ .Form__hint--required {
+.Form:not(.Form--serverError):not(.Form--justSend):not(.Form--loaded) .Form__queryTextArea:focus-visible:valid ~ .Form__hint--progress {
   opacity: 100%;
 }
 
-.Form:not(.Form--serverError):not(.Form--justSend) .Form__queryTextArea:focus-visible:valid ~ .Form__hint--progress {
+.Form--loaded .Form__loader {
   opacity: 100%;
 }
+
+.Form__loader::after {
+  display: inline-block;
+  width: 1em;
+  height: 1em;
+  margin-left: 1ch;
+  content: "";
+  background-image: radial-gradient(var(--main-bg) 60%, transparent 60%), conic-gradient(transparent, currentcolor);
+  border-radius: 50%;
+  animation-name: form-loader;
+  animation-duration: 1s;
+  animation-iteration-count: infinite;
```

```suggestion
  animation-iteration-count: infinite;
  animation-timing-function: linear;
```

It will make animation more smooth. Without iteration with the stop in the middle.

---

I think not linear is more... beautiful. I did it on purpose.

---

Let’s ask designers

---

Non-linear is more interesting, but we could refine the timing for this option to smooth out the pause. Now it is really noticeable. But in general, everything is good!

---

We can try `cubic-bezier(0.12, 0, 0.39, 0)`?

https://easings.net/#easeInSine

The default easings it really stopped in the end https://cubic-bezier.com/#.25,.1,.25,1

---

@apspace what do you think?
https://user-images.githubusercontent.com/27442574/183611129-bebd556e-f47d-43e7-b917-0083676283ab.mov

---

We agreed on timing function that i just pushed.

---

Resolve merge conflict and merge the PR